pages/price-feeds/use-real-time-data/sui.mdx

```diff
@@ -77,7 +77,7 @@ const priceUpdateData = await connection.getPriceFeedsUpdateData(priceIds); // s
 // It is either injected from browser or instantiated in backend via some private key
 const wallet: SignerWithProvider = getWallet();
 // Get the state ids of the Pyth and Wormhole contracts from
-// https://docs.pyth.network/documentation/pythnet-price-feeds/sui#contracts
+// https://docs.pyth.network/price-feeds/contract-addresses/sui
```

Comment:
Sui page loads slowly

Reply:
yeah it's been like this. I think it's something to do with some of the formatting, but i'm not sure.